cni/netconfig.go

```diff
@@ -56,6 +56,7 @@ type NetworkConfig struct {
 	EnableSnatOnHost           bool     `json:"enableSnatOnHost,omitempty"`
 	EnableExactMatchForPodName bool     `json:"enableExactMatchForPodName,omitempty"`
 	CNSUrl                     string   `json:"cnsurl,omitempty"`
+	HairpinOffOnHostInterface  bool     `json:"hairpinOffOnHostInterface,omitempty"`
```

Comment:
can we define this above CNSUrl so that all bools will be in one place.

Instead of HairpinOffOnHostInterface  can we name it as DisableHairpinOnHostInterface  ? Its' upto you.

Reply:
Updated the order.

I went back & forth between DisableHairpin*** vs HairpinOff***. Finally settled on the current name because this flag isn't actually "disabling" the hairpin on the interface, it is never enabling it.

Reply:
Updated the ordering of the parameters.

I'll prefer keeping the name as HairpinOff***. I had considered Disable*** earlier, but this feature isn't really disabling hairpin instead it is not enabling it :).

Reply:
flag name is super confusing. It should be `DisableHairpin..` with default false.

Reply:
@vipulha the argument goes same for hairpinoff. If this is true, its setting that to off instead its not turning it on.

Reply:
Renamed the parameter.